idiss/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "idiss"
 build = "build.rs"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
There is no need to change this version. This functionality is not affected.

Reply:
The `cs_export.rs` has changed and that is actually a breaking change which I didn’t mention in the changelog. I have updated the changelog now and bump the major.

Please let me now if procedure is wrong.